generators/app/templates/infra/gcp/cloudrun.tf

```diff
@@ -0,0 +1,24 @@
+# This will just create a default Cloud Run service but 
+# it will not update and mangae your revisions as Terraform is not meant to do this, that's a CD job.
+
+resource "google_cloud_run_v2_service" "this" {
+  name     = "${var.api_name}-service-${terraform.workspace}"
+  ingress  = "INGRESS_TRAFFIC_ALL"
```

Comment:
Maybe we should warn that it opens the API to anyone on the internet, and that some security rules should be set up.

Reply:
As seen together, there is default authentication